### ISSUE TITLE
After #305, ensure we remove unnecessary dependencies from `analytics`

### ISSUE THREAD

**Issue by nathan-at-least** [LeastAuthority/leastauthority.com#306]
Ticket #305 is about improving the monitoring dependency management so that we don't need unnecessary dependencies.  This ticket is about removing the dependencies we've installed manually.

Here are a set of Tahoe dependencies we installed on analytics which are unnecessary for monitoring:

```
daira@ip-10-204-239-9:~$ sudo apt-get install libffi6 openssl libssl1.0.0 python-nevow python-crypto python-dateutil python-foolscap python-six python-pycparser python-unidecode python-zfec python-simplejson
```

**Comment by exarkun** [comment]
The entire monitoring system fell over a long time ago, unfortunately. We have a "Monitoring and Reporting" milestone. We'll put plans for future work there.
